modules/gltf/gltf_document.cpp

```diff
@@ -5348,7 +5353,7 @@ void GLTFDocument::_convert_csg_shape_to_gltf(CSGShape3D *p_current, GLTFNodeInd
 	GLTFMeshIndex mesh_i = p_state->meshes.size();
 	p_state->meshes.push_back(gltf_mesh);
 	p_gltf_node->mesh = mesh_i;
-	p_gltf_node->transform = csg->get_meshes()[0];
+	p_gltf_node->transform = csg->get_transform();
```

Comment:
Weird: i wonder why `get_meshes()[0]` is hardcoded to always return Transform3D::IDENTITY